packages/flame_oxygen/LICENSE

```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2021 FlameEngine
```

Comment:
We are using blue fire on the other licenses I believe

Reply:
Forgot to change it, updated.